soc/arm64/intel_socfpga/agilex5/Kconfig.defconfig.agilex5

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_AGILEX5
+
+config SOC
+	default "intel_socfpga_agilex5"
+
+# must be >= the highest interrupt number used
+# - include the UART interrupts 173 or 204
+config NUM_IRQS
+	int
+	default 205
```

Comment:
update to value above 242

Reply:
Updated now as minimal required for initial Agilex5 BSP. 
Will update this as and when required drivers are implemented.